react/Header/UserAccountMenu/UserAccountMenu.less

```diff
@@ -76,25 +76,33 @@
   }
 }
 
-.icon.jobSearch { color: @sk-blue-lighter; }
+.icon.jobSearch { color: @sk-teal; }
 .icon.profile { color: @sk-purple; }
 .icon.saveJobs { color: @sk-pink; }
-.icon.saveSearches { color: @sk-blue-lighter; }
+
+.icon.saveSearches {
+  color: @sk-blue-lighter;
+  @media @mobile {
+    color: @sk-highlight;
```

Comment:
Curious why this colour is different on mobile? Also it would be preferred to use a blue from the palette rather than cross use the `highlight` colour if possible.

Reply:
I think Carmelia was saying the there's no blue that she'd prefer here instead, but I'll doublecheck. Difference is because works differently on dark and light background, this is the only responsive blue icon